packages/bbui/src/Form/Core/CheckboxGroup.svelte

```diff
@@ -25,6 +30,8 @@
   }
 </script>
 
+import context from "src/context"
```

Comment:
Not sure how this got here, but I don't think it should be!

Reply:
Huh, neither do I. Almost certainly auto-added by VSCode accident. 🙈